### PR TITLE
Collection of fixes for Flathub submission

### PR DESCRIPTION
**Pull request by Siosm**

### CODE REVIEW

File: packaging/org.freeorion.FreeOrion.metainfo.xml [hunk 1]
```diff
@@ -38,4 +38,21 @@
       <image width="1600" height="900" type="source">https://www.freeorion.org/screenshots/FreeOrion_ResearchScreen.png</image>
```
[the remark this answers is not in the document]
Comment (Siosm) [reply packaging/org.freeorion.FreeOrion.metainfo.xml:38]: I gave this a try with the last push.

[the remark this answers is not in the document]
Comment (Siosm) [reply packaging/org.freeorion.FreeOrion.metainfo.xml:38]: Split into: https://github.com/freeorion/freeorion/pull/3154

**Comment by Siosm** [comment]
I have updated this PR to add the violence rating and split the screenshot request into another PR. This should be ready for another review.

File: packaging/org.freeorion.FreeOrion.metainfo.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vezzra) [reply packaging/org.freeorion.FreeOrion.metainfo.xml:38]: See my comment [here](https://github.com/freeorion/freeorion/pull/3154#issuecomment-680796958).

**Comment by geoffthemedio** [comment]
> I have updated this PR to add the violence rating

I had a look at the [OARS site](https://hughsie.github.io/oars/generate.html), and I'm really uncertain about how to apply the ratings to FreeOrion... Or rather, they mostly don't seem to apply.

For example, it refers to graphic depictions of "fictional characters depicted in an animated film or a comic strip" or "realistic characters" or "fantasy characters" in situations / doing things.

FreeOrion just has text saying, in effect, "there was a battle here" with some numbers. There are certainly no "characters" involved.

The wording of "bloodshed" is more general, but at most "mild" or "unrealistic bloodshed" seems applicable.

The discrimination categeory might apply, though it's not clear if "race" applies to fantasy but sentient "species". There are "Concentration Camps" in-game so it's probably a fair label to apply, even if it's seemingly out of the context the categorization was intended for.

So maybe
````
  <content_rating type="oars-1.1">
    <content_attribute id="violence-bloodshed">mild</content_attribute>
    <content_attribute id="language-humor">mild</content_attribute>
    <content_attribute id="language-discrimination">intense</content_attribute>
    <content_attribute id="social-chat">intense</content_attribute>
  </content_rating>
````

**Comment by Siosm** [comment]
Thanks for the context. I wasn't really sure myself how much to rate each. Updated with your changes.

**Pull request merged**